data/35e/wizards_of_the_coast/supplement/monster_manual_3/mm3_races.lst

```diff
@@ -3,6 +3,8 @@
 #Ambush Drake	SIZE:M	RACETYPE:Dragon	MONSTERCLASS:Dragon:7	RACESUBTYPE:Drake	MOVE:Walk,40,Fly,30	NATURALATTWACKS:Bite,Natural.Weapon.Piercing,*1,1d8|Claws,Natural.Weapon.Slashing,*2,1d6		ABILITY:Internal|AUTOMATIC|Race Traits ~ Ambush Drake
 #scent, spell resistance 16, telepathic link 30 ft.
 
+Flind Gnoll		RACETYPE:Humanoid	RACESUBTYPE:Gnoll	TYPE:Humanoid	LEGS:2	SIZE:M	MOVE:Walk,30	FACE:5	REACH:5		FAVCLASS:Ranger		STARTFEATS:1	AUTO:LANG|Gnoll|Common		LANGBONUS:Draconic,Elven,Goblin,Orc	CHANGEPROF:Flindbar=Martial	BONUS:COMBAT|AC|2|TYPE=NaturalArmor	BONUS:STAT|STR|6	BONUS:STAT|CON|4	BONUS:STAT|DEX|2	BONUS:VAR|DarkvisionRange|60|TYPE=Base		AUTO:ARMORPROF|ARMORTYPE=Light		AUTO:SHIELDPROF|SHIELDTYPE=Buckler|SHIELDTYPE=Heavy|SHIELDTYPE=Light	AUTO:WEAPONPROF|Battleaxe|Shortbow|TYPE=Simple			ABILITY:Internal|AUTOMATIC|Race Traits ~ Gnoll		MONCSKILL:Listen|Spot		MONSTERCLASS:Humanoid (Fortitude):2		LEVELADJUSTMENT:2	CR:2			SOURCEPAGE:62
```

Comment:
Some extras for consideration in the future, otherwise excellent job.
Name  'Flind' instead of 'Flind Gnoll'
KEY:Gnoll (Flind)
SORTKEY:Gnoll